lib/logstash/codecs/netflow.rb

```diff
@@ -33,13 +33,17 @@ class LogStash::Codecs::Netflow < LogStash::Codecs::Base
   # See <https://github.com/logstash-plugins/logstash-codec-netflow/blob/master/lib/logstash/codecs/netflow/netflow.yaml> for the base set.
   config :definitions, :validate => :path
 
+  NETFLOW5_FIELDS = ['version', 'flow_seq_num', 'engine_type', 'engine_id', 'sampling_algorithm', 'sampling_interval', 'flow_records']
+  NETFLOW9_FIELDS = ['version', 'flow_seq_num']
+  SWITCHED = /_switched$/
+  FLOWSET_ID = "flowset_id"
+
   public
```

Comment:
I agree and if you look at the changes, you'll notice I cleaned all the useless `public` statements, this one is a left-over. typically separating the `public` from the `private` only take a single `private` statement.

It is certainly not an error to put `public` / `private` for every method, I am not sure why this pattern was adopted in the first place but I assume it was to be very explicit for every method. I personally find it too verbose but that's my taste.